app/public/dist/client/locales/en/translation.json

```diff
@@ -1256,383 +1256,383 @@
+    "TAIL_GLOW": "Deal [30,SP] SPECIAL to adjacent enemy Pokémon and increase the user's AP by 50%.",
+    "STRUGGLE_BUG": "Deal [30,SP] SPECIAL to adjacent enemy Pokémon and reduce their AP by 50%.",
+    "PRISMATIC_LASER": "The user shoots a prismatic laser in a large straight line, dealing [80,SP] SPECIAL and displacing any enemy Pokémon in its path.",
+    "NATURAL_GIFT": "Heal the most injured allied Pokémon for [30,60,120,SP] HP and grant them RUNE_PROTECT for [1,2,3] seconds.",
+    "NIGHT_SHADE": "Deal [25,33,50,SP=0.5]% of target's max HP as TRUE.",
+    "CHARGE_BEAM": "Fire a concentrated bundle of electricity that propagates to a maximum of 3 adjacent enemy Pokémon, dealing [15,30,60,SP] SPECIAL to each.",
+    "POPULATION_BOMB": "Deal 10 SPECIAL [4,8,12,16,SP] times to the target.",
+    "SCREECH": "Reduce DEF of all adjacent enemy Pokémon by [1,2,4,SP].",
+    "SAND_TOMB": "Deal [10,20,40,SP] SPECIAL to the target and inflict them with SILENCE and PARALYSIS for [3,5,8] seconds.",
+    "WHIRLWIND": "Deal [40,80,120,SP] SPECIAL to the target and make them fly away.",
+    "EMPTY_LIGHT": "Spit poison directly at an enemy Pokémon, dealing [33,SP] SPECIAL and reducing their SPE_DEF by 3. The ability then rebounds to another enemy Pokémon with a lower SPE_DEF. (5 bounces maximum)",
+    "UNBOUND": "Unbinds itself to become Hoopa's true form and gains a humongous amount of power.",
+    "HYPERSPACE_FURY": "Using its many arms, unleashes a barrage of [4,SP] attacks. Each attack deals 15 SPECIAL to the target and lowers their DEF by 1.",
+    "SNIPE_SHOT": "Fire a shot of water from the user's index fingertips at the farthest enemy Pokémon, dealing [50,100,200,SP] SPECIAL to all enemy Pokémon in its path.",
+    "AIR_SLASH": "Deal [25,50,100,SP] SPECIAL to the target and inflict them with FLINCH for 7 seconds.",
+    "EGGSPLOSION": "Launch an explosive egg at the target, dealing [40,80,140,SP] SPECIAL to all adjacent enemy Pokémon and inflicting them with ARMOR_REDUCTION for 4 seconds. For each enemy Pokémon KO'd, there is a [30,LK]% chance of an egg appearing on the player's bench.",
+    "FLORAL_HEALING": "Heal HP equal to max PP of Comfey's holder or Comfey itself if not held.",
+    "VINE_WHIP": "Deal [100,SP] SPECIAL to the target and cause them and all adjacent enemy Pokémon to FLINCH for 3 seconds.",
+    "BODY_SLAM": "Deal [30,SP]% of the user's max HP as SPECIAL to the target.",
+    "BARB_BARRAGE": "Dive into the middle of the enemy team, dealing [15,30,60,SP] SPECIAL to all adjacent enemy Pokémon and inflict them with POISONNED for 3 seconds.",
+    "MAGIC_POWDER": "Gain [10,20,40,SP] SHIELD and SILENCE adjacent enemy Pokémon for [2,3,4] seconds.",
+    "INFERNAL_PARADE": "Shoot a returning Flame Wheel that has a [50,LK]% chance to apply BURN for 3s and dealing [30, SP] SPECIAL to all enemy Pokémon in its path. May hit twice.",
```

Comment:
Why "may hit twice" ? It always hit twice, it applies 2 attacks

Reply:
The original description makes it sound like a projectile with travel time, that does one attack going out and one more attack going back in. Does the attack happen twice instantly? Either way the wording is weird. It should probably be more like "travels its path twice"

Reply:
ability_description.WHEEL_OF_FIRE has the same issue.

Reply:
the animation delay is short enough for this not to be a concern

Reply:
Thanks for the clarification. Fixed